contributing/tests.md

```diff
@@ -126,24 +126,24 @@ The important args are:
 
 - The first, optional, parameter is a location to a folder that will serve as
   the workspace to run the tests. If you omit this, it just uses a clean
-<<<<<<< HEAD
```

Comment:
Looks like this file wasn't properly cleaned up in a previous merge.

Reply:
This is just part of the tests.md doc but it's incredible confusing when reviewing it :)
![screen shot 2019-02-19 at 2 33 52 pm](https://user-images.githubusercontent.com/1041842/53052394-787e8c00-3453-11e9-8ef8-b120ddb00a71.png)

Reply:
It looks like there was a three-way merge unrelated to this PR in https://github.com/forcedotcom/salesforcedx-vscode/commit/7ff38a9b7c7f4ed8f887f4d5f14c026c74249389#diff-7f9d343d396a3b97d4b2960c77dfb89aR83 that wasn't resolved properly. The `contributing/tests.md` is a bit broken even in the develop branch https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/contributing/tests.md, but that can be fixed in a separate PR.

Reply:
I see what you mean. I'll just fix it here, thanks for catching that 👍